### ISSUE TITLE
Stars don't come out at night

### ISSUE THREAD

**Issue by rockwalrus** [supermedium/aframe-environment-component#8]
The stars are not visible with the latest code in Chrome or Firefox on my machine, a Macbook with Intel HD 5000 graphics. To reproduce, I just have to go to the "starry" preset in the demo application.

**Comment by feiss** [comment]
I will try it on a macbook asap. Stars are very very small on purpose so in VR mode they feel right. Maybe in your screen they are rendered too small to preceive but they are ok in vr mode? thanks!

**Comment by stevel4857** [comment]
It works great for me. 1080 card.

**Comment by feiss** [comment]
Oh, I thought mine had an Intel. It's an old macbook, GeForce 9400M, works fine.

**Comment by rockwalrus** [comment]
Yeah, I can just barely see them when I switch to a lower-DPI display. Maybe the star size could be configurable, or somehow adapt to the display.

**Comment by feiss** [comment]
hmm.. the important scenario is when you are in VR, when they have to look nice. I'll think about what would be the best solution.. Configurable size does not seem very useful.. Do they look ok in-VR for you? are you making a only-2d experience? Thanks!

**Comment by rockwalrus** [comment]
The demo I'd like to use this for is supposed to highlight the versatility of WebVR, so it's designed to work on 2D desktop, Google Cardboard, and HTC Vive. There's a good chance we'll support Daydream and GearVR too.  We're currently using a texture map for the stars in our scene, but it's not corrected for distortion, so it's a bit ugly at the zenith.

**Comment by feiss** [comment]
I see. Thanks for sharing. My concern now is that stars are currently squares, so if you make them bigger they will look bad. I'll try to use small rounded sprites and add a size parameter.

**Comment by rockwalrus** [comment]
I wonder if it's possible to auto-adjust the size based on the DPI of the device.

**Comment by feiss** [comment]
is this still needed?
